src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs

```diff
@@ -504,10 +504,7 @@ internal static RemoteSessionNamedPipeServer CreateRemoteSessionNamedPipeServer(
                 securityAttributes);
 
             int lastError = Marshal.GetLastWin32Error();
-            if (securityDescHandle != null)
-            {
-                securityDescHandle.Value.Free();
-            }
+            securityDescHandle?.Free();
```

Comment:
It generates more efficient code. I wonder but it is great!
Compere:
[before](https://sharplab.io/#v2:C4LghgzgtgPgAgJgIwFgBQcAMACOSB0ASgK4B2wAllAKb4CS51ATgPYAOAyswG4UDG1CAG50WXAS59iTCsACe+AIJ8BECAGEW5VgBsRaUQGZcCbOuwBvdNhu5jcACzYAsgApNUKFsnTZcgCKCfDJswCxM2BDUUjLygRB8AJSW1rZpAOLqABJgpAAmOtQA/JHRvnFBOfmF2AC82KTEOnqpaTYUAGbYrlExfvF82ACE9Y3Nia1tVmhts9gARnLA1ADaALqlfRUJAELEHR3MdQ3UAO4LS6u95QFB+DsUpGBMcgAy1KQA5sAAFmv6cza11itwS+HS1GADyeLwAYuEoD0yiCBnsDswADTYTCJAGA2zA/qVXIFajHTJVUlKZosPhIragvhow5MLEUkmFAAqcjYtAACo9SNQ8rjJmkAL5i2xS9pdek3AaUmojBpNHQTGb4mzTLWzQnbPhK2gANTAOmItFhTGo1Fcos1WslDuwTvFQA=)
[after](https://sharplab.io/#v2:C4LghgzgtgPgAgJgIwFgBQcAMACOSB0ASgK4B2wAllAKb4CS51ATgPYAOAyswG4UDG1CAG50WXAS59iTCsACe+AIJ8BECAGEW5VgBsRaUQGZcCbOuwBvdNhu5jcACzYAsgApNUKFsnTZcgCKCfDJswCxM2BDUUjLygRB8AJSW1rZpAOLqABJgpAAmOtQA/JHRvnFBOfmF2AC82KTEOnqpaTYUAGbYrlExfvF82ACE9Y3Nia1tVmhts9gARnLA1ADaALqlfRUJAELEHR3MdQ3UAO4LS6u95QFB+DsUpGBMcgAy1KQA5sAAFmv6cza11itwS+HS1GADyeLwAYuEoD0yiCBnsDswADTYTCJAGA2zA/qVXIFajHTJVUlKZosPhIragvhow5MLEUkmFAAqcjYtAACo9SNQ8rjJmkAL5i2xSmyE7Z8SmFIr4WFMajUVyima2SVocVAA===)